Gopkg.toml

```diff
@@ -0,0 +1,23 @@
+[[constraint]]
+  branch = "master"
+  name = "github.com/container-storage-interface/spec"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/golang/glog"
+
+[[constraint]]
+  name = "google.golang.org/grpc"
+  version = "1.7.2"
+
+[[constraint]]
+  name = "github.com/docker/distribution"
+  revision = "edc3ab29cdff8694dd6feb85cfeb4b5f1b38ed9c"
+
+[[constraint]]
+  name = "k8s.io/kubernetes"
+  version = "v1.9.0-beta.2"
```

Comment:
see if you can get 1.9.1, it is there now https://github.com/kubernetes/kubernetes/tree/v1.9.1
(also apimachinary)

Reply:
done